xml/operations-configuring-identity-websso.xml

```diff
@@ -52,8 +52,17 @@
    </listitem>
    <listitem>
     <para>
-     Only WebSSO federation using the SAML method is supported in
-     &product; . OpenID-based federation is not currently supported.
+     The &productname; WebSSO function with OpenID method was tested with
+     Google OAuth 2.0 APIs, which conform to the OpenID Connect specification.
+     The interaction between Keystone and the external Identity Provider (IdP) is
+     handled by the Apache2 <link xlink:href="https://github.com/zmartzone/mod_auth_openidc">auth_mod_openidc</link> module. Please consult with the specific OpenID Connect vendor
+     on whether they support <link xlink:href="https://github.com/zmartzone/mod_auth_openidc">auth_mod_openidc</link>.
```

Comment:
format like similar content in `operations-configuring-identity-identity_limitations.xml` above.

Reply:
Done